packages/v4/patternfly-docs/content/accessibility/badge/badge.md

```diff
@@ -3,30 +3,61 @@ id: Badge
 section: components
 ---
 
-A **badge** is used to annotate other information like a label or an object name. Badges are typically used to reflect counts, like number of unread notifications. Badges are not interactive.
+import { Checkbox, List, ListItem } from '@patternfly/react-core';
 
-**Keyboard users** should not be able to focus on a badge.
+## Accessibility
 
-**Screen reader users** should be able to have a screen reader describe the contents of and purpose of a badge when it pops up, but not focus on it otherwise.
+To implement an accessible PatternFly **badge**:
 
-No props need to be added or modified for badge accessibility.
+- Ensure the badge itself is not interactive in anyway. 
+- Ensure that the surrounding context can convey to users via screen reader or similar assistive technologies the purpose of the badge.
```

Comment:
Rearrange to make the sentence easier to read (bold just showing the change)- Ensure that the surrounding context can convey **the purpose of the badge** to users via screen reader or similar assistive technologies.